wandb/run_manager.py

```diff
@@ -925,6 +925,12 @@ def init_run(self, env=None):
                 # Currently we assume we're not resuming in the case of resume = auto,
                 # and we throw an error in the case of resume = must.
                 logger.info("checking resume status, waiting at most %d seconds" % InternalApi.HTTP_TIMEOUT)
+
+                if not self._project:
+                    raise LaunchError(
+                        "resume='must' but no project is currently specified, pass in `project='...'` to set up this "
```

Comment:
Sorry for the back and forth, let's do:

`"resume='must' but no project is specified.  Pass project to init: wandb.init(project="...")"`